### PR TITLE
Ml pipeline rewrite

### PR DESCRIPTION
**Pull request by shailx383**
-   **Please check if the PR fulfills these requirements**

*   [x] The commit message follows our guidelines
*   [x] Tests for the changes have been added (for bug fixes / features)
*   [ ] Docs have been added / updated (for bug fixes / features)
*   [x] The code runs on the contributors machine.
*   [x] The code follows the code style guide of this project.

-   **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
refactor 
-   **What is the current behavior?** (You can also link to an open issue here)
pipeline works with inception resnet
-   **What is the new behavior (if this is a feature change)?**
included FaceNet from model.py
-   **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no
-   **Other information**:

### CODE REVIEW

**Comment by adityapentyala** [comment]
Works when commit 64c5cdf is reverted, with the exception of removing `points.data[0].numpy()` in favour of `points.data.numpy()` on line 90. Check if it works after changes. The rest of it works perfectly so I'll push if the change works

**Review by adityapentyala — CHANGES_REQUESTED** [review]
also, could you add tqdm to the env file? had to install that but again it might be a problem with only my env

**Comment by shailx383** [comment]
removed tqdm as it is not required

**Review by adityapentyala — APPROVED** [review]

File: clusterer/datapoint_clusterer.py
Comment (adityapentyala) [inline clusterer/datapoint_clusterer.py:1]: line 90 still throws an error when only `point.data` is written. code runs perfectly with `point.data.numpy()`

**Review by adityapentyala — CHANGES_REQUESTED** [review]

**Review by adityapentyala — APPROVED** [review]
LGTM, merging

**Pull request merged**